src/NXP/Classes/Calculator.php

```diff
@@ -77,7 +77,7 @@ public function calculate(array $tokens, array $variables)
         }
         $result = array_pop($stack);
         if ($result === null || !empty($stack)) {
-            throw new IncorrectExpressionException();
+            throw new IncorrectExpressionException('Empty Stack');
```

Comment:
Here stack must be empty at end of calculation :) 
```suggestion
            throw new IncorrectExpressionException('Stack must be empty');
```